### PR TITLE
bug 1219813, 1219818 - use djcelery_transactions for emails

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: kuma/users/tasks.py [hunk 1]
```diff
@@ -8,6 +7,8 @@
 from django.core.mail import EmailMultiAlternatives
 from django.template.loader import render_to_string
 
+from djcelery_transactions import task
```
Comment (robhudson) [inline kuma/users/tasks.py]: How about `from djcelery_transactions import task as transaction_task` instead?

**Comment by groovecoder** [comment]
Updated to make both usages `from djcelery_transactions import task as transaction_task`.

**Comment by jwhitlock** [comment]
Need to add ``CELERY_EAGER_TRANSACTION = True`` to settings (see https://github.com/fellowshipofone/django-celery-transactions#compatibility-with-celery_always_eager), which then makes tests fail :cry:

**Comment by jwhitlock** [comment]
+r.  On local dev box, both emails are sent async, without the errors in the bug report.  ``CELERY_EAGER_TRANSACTION = True`` was **not** used.

**Pull request merged**